src/motor-positioning/@GeneticAlgorithm/updateRouletteWheel.m

```diff
@@ -0,0 +1,8 @@
+function updateRouletteWheel(obj)
```

Comment:
roulette wheel? :-D

Reply:
Fixed in https://github.com/ami-iit/paper_bergonti_2022_tro_kinematics-control-morphingcovers/commit/f8b61d5d2560a1af5356fa5d3b4de5acbe74a349